<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COVID19_Updater</class>
 <widget class="QMainWindow" name="COVID19_Updater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>917</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>917</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Live COVID-19 Updater</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 25, 55);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	
	
	background-color: rgb(5, 25, 55);

	border-radius: 10px
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Title">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>20</y>
         <width>921</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(3, 122, 245);
lable.adjustSize();</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;COVID-19&lt;/strong&gt; &lt;i&gt;LIVE&lt;/i&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="Description">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>70</y>
         <width>921</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 191, 196);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Realtime COVID-19 status of all Countries including &lt;span style=&quot; font-weight:600;&quot;&gt;Indian&lt;/span&gt; States and Districts&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="AuthorName">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>570</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(190, 191, 196,95);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Created by: &lt;span style=&quot; font-style:italic;&quot;&gt;Sumukh Jadhav&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QPushButton" name="Confirmed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica LT Std</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(112,108,195,50);

color: rgb(112,108,195);


</string>
       </property>
       <property name="text">
        <string>Confirmed
 - - -</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Active">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>460</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica LT Std</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgba(238, 14, 63,50);

color: rgb(238, 14, 63);


</string>
       </property>
       <property name="text">
        <string>Active
 - - -</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Recovered">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>460</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica LT Std</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(46, 155, 80,50);
color: rgb(42, 160, 79);


</string>
       </property>
       <property name="text">
        <string>Recovered
 - - -</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Deaths">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>460</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica LT Std</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(110, 116, 128,50);
color: rgb(110, 114, 126);

</string>
       </property>
       <property name="text">
        <string>Deaths
 - - -</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="UpdataData_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>420</y>
         <width>901</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gravity</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">

color: rgb(169, 206, 244);
border-radius: 30px;

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="UpdataData_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>350</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Emoji</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgba(0, 122, 255,50);
	color: rgb(3, 122, 245);
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgba(0, 122, 255,45);

}
QPushButton:pressed {
	background-color: rgba(0, 122, 255,40);

}</string>
       </property>
       <property name="text">
        <string>New Cases</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>220</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
border:none;
color: rgba(190, 191, 196,150);
border-bottom:1px solid #717072;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string> District</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>280</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
border:none;
color: rgba(190, 191, 196,150);
border-bottom:1px solid #717072;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Number of Cases</string>
       </property>
      </widget>
      <widget class="QLabel" name="Title_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>901</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2f85dc;
lable.adjustSize();</string>
       </property>
       <property name="text">
        <string>&lt;i&gt;&lt;strong&gt;Update Status&lt;/strong&gt; &lt;/i&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="UpdataData_4">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>350</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Emoji</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgba(0, 122, 255,50);
	color: rgb(3, 122, 245);
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgba(0, 122, 255,45);

}
QPushButton:pressed {
	background-color: rgba(0, 122, 255,40);

}</string>
       </property>
       <property name="text">
        <string>Recovery</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="UpdataData_5">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>350</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Emoji</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgba(0, 122, 255,50);
	color: rgb(3, 122, 245);
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgba(0, 122, 255,45);

}
QPushButton:pressed {
	background-color: rgba(0, 122, 255,40);

}</string>
       </property>
       <property name="text">
        <string>Deaths</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="Title_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>540</y>
         <width>901</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 14, 63);
lable.adjustSize();</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="newConfirmed">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>540</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(112,108,195);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="newActive">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>540</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(238, 14, 63);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="newRecovered">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>540</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(42, 160, 79);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="Title_7">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>540</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(110, 114, 126);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>Title_3</zorder>
      <zorder>Title</zorder>
      <zorder>Description</zorder>
      <zorder>AuthorName</zorder>
      <zorder>Confirmed</zorder>
      <zorder>Active</zorder>
      <zorder>Recovered</zorder>
      <zorder>Deaths</zorder>
      <zorder>UpdataData_2</zorder>
      <zorder>UpdataData_3</zorder>
      <zorder>lineEdit</zorder>
      <zorder>lineEdit_2</zorder>
      <zorder>Title_2</zorder>
      <zorder>UpdataData_4</zorder>
      <zorder>UpdataData_5</zorder>
      <zorder>newConfirmed</zorder>
      <zorder>newActive</zorder>
      <zorder>newRecovered</zorder>
      <zorder>Title_7</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
